WordPress/Classes/ViewRelated/Aztec/ViewControllers/AztecPostViewController.swift

```diff
@@ -3576,4 +3587,9 @@ extension AztecPostViewController {
         static let acceptTitle  = NSLocalizedString("Yes", comment: "Yes")
         static let cancelTitle  = NSLocalizedString("Not Now", comment: "Nicer dialog answer for \"No\".")
     }
+
+    struct MediaUnableToPlayVideoAlert {
+        static let title = NSLocalizedString("Unable to play video", comment: "Dialog box title for when the user is cancelling an upload.")
+        static let message = NSLocalizedString("We are unable to get information from this video at the moment. Please try later.", comment: "This prompt is displayed when the user attempts to play a video in the editor but for some reason we are unable to retrieve from the server.")
```

Comment:
I'm not quite sure about the wording of "We are unable to get information from this video at the moment." – perhaps just something more generic like "Something went wrong. Please try again."?